lib/index.ts

```diff
@@ -8,6 +8,10 @@ const windowregistry = process.platform === 'win32' ? require('../build/Release/
 export type HKEY = "HKEY_CURRENT_USER" | "HKEY_LOCAL_MACHINE" | "HKEY_CLASSES_ROOT" | "HKEY_USERS" | "HKEY_CURRENT_CONFIG";
 
 export function GetStringRegKey(hive: HKEY, path: string, name: string): string | undefined {
+	if (process.platform !== 'win32') {
+		throw new Error('GetStringRegKey is only available on Windows.');
+	}
+
 	if (windowregistry) {
 		return windowregistry.GetStringRegKey(hive, path, name);
 	}
```

Comment:
```suggestion
	if (windowregistry) {
		return windowregistry.GetStringRegKey(hive, path, name);
	}
	
	throw new Error('GetStringRegKey is only available on Windows.');
```

Reply:
There's more code below in case the module for some reason cannot be initialized. The platform if statement differentiates that case from being on a non-Windows platform.

Reply:
`require` is synchronous, if it failed it would throw before this block got executed.